src/azure-cli/HISTORY.rst

```diff
@@ -37,6 +37,7 @@ Release History
 
 * Azure Stack: surface commands under the profile of 2019-03-01-hybrid
 * functionapp: Add ability to create Java function apps in Linux
+* Add az webapp|functionapp config ssl create-managed-certificate
```

Comment:
Please make your release note in next release part.

Reply:
removed the History.rst release note as this is extracted from the title now